plugins/inputs/logfile/fileconfig_test.go

```diff
@@ -233,6 +233,17 @@ func TestTimestampParserWithFracSeconds(t *testing.T) {
 		fmt.Sprintf("The timestampFromLogLine value %v is not the same as expected %v.", timestamp, expectedTimestamp))
 }
 
+func TestNonAllowlistedTimezone(t *testing.T) {
+	fileConfig := &FileConfig{
+		Timezone: "EST",
```

Comment:
may be a dumb question - Will this fail if the test is run outside of the EST?

Reply:
You can input any value and it will always pass since we always default to Local when timezone is not specified as "UTC"